docs/src/main/paradox/overview.md

```diff
@@ -0,0 +1,38 @@
+# Overview
+
+## gRPC
+
+[gRPC](https://grpc.io) is a transport mechanism for request/response and (non-persistent) streaming use cases. See
+@ref[Why gRPC?](whygrpc.md) for more information about why to choose gRPC as your RPC transport.
```

Comment:
Maybe 'about when to use gRPC as your transport', but that's nitpicking.

Reply:
Thanks, done.